interfaces/console/ui/layout/Header/Alert/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { useUpdateNotificationMutation } from '@/generated';
+import { NotificationsResDto } from '@/generated/metrics';
+import { colors } from '@/styles/theme';
+import AlertBox from '@/ui/molecules/AlertBox';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import { Badge, IconButton, Popover } from '@mui/material';
+import React, { useState } from 'react';
+
+const IconStyle = {
+  '.MuiSvgIcon-root': {
+    width: '24px',
+    height: '24px',
+    fill: colors.white,
+  },
+  '.MuiBadge-root': {
+    '.MuiSvgIcon-root': {
+      width: '24px',
+      height: '24px',
+      fill: colors.white,
+    },
+  },
+};
+
+interface IAlertsProps {
+  alerts: NotificationsResDto[] | undefined;
+  setAlerts: Function;
+}
+
+const Alerts = ({ alerts, setAlerts }: IAlertsProps) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [updateNotificationMutation] = useUpdateNotificationMutation();
+
+  // Handle popover open
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  // Handle popover close
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  // Mark alert as read
+  const handleAlertRead = (index: number) => {
+    if (alerts) {
+      let alertId = alerts[index].id;
+      updateNotificationMutation({
+        variables: {
+          updateNotificationId: alertId,
+          isRead: true,
+        },
+      });
```

Comment:
We try to keep our components dumb, All state management and network call handled at page level.